### ISSUE TITLE
No moderator options in open groups. 

### ISSUE THREAD

**Issue by Lucifer1903** [oxen-io/session-android#791]
**Code of conduct**

- [ x ] I have read and agree to adhere to the [Code of Conduct](https://github.com/oxen-io/session-android/blob/master/CODE_OF_CONDUCT.md).

**Describe the bug**

Moderator options not present when accessing open groups from android. 

**To reproduce**

Create an open groups, set user as moderator, use desktop to make sure user has moderator options, use Android and moderator options will be missing. 

**Smartphone (please complete the following information):**

 - Device: Huawei 
 - OS: Android 10
 - Version of Session: 1.11.12 (2315)

**Additional context**

A few months ago moderator options were available in android. I don't know why they are missing now.

**Comment by ianmacd** [comment]
Yes, even adding a group avatar seems to be impossible now.

**Comment by hjubb** [comment]
I agree the moderation options from mobile at the moment is currently pretty basic, will look at incorporating more options that make sense shortly. Is there any you or any other open group moderators would see as a priority to be accessible on mobile?

**Comment by Lucifer1903** [comment]
It's not that moderation options from mobile are basic. It's that they used to be present (changing group name and image, and deleting other users messages), however now there is no difference between moderators and standard users. 

If someone is added as a moderator and they only use android then they wouldn't even know that they are a moderator as there would be no noticeable difference. However other users would be able to see the crown on the moderators icon.

**Comment by ianmacd** [comment]
> I agree the moderation options from mobile at the moment is currently pretty basic, will look at incorporating more options that make sense shortly. Is there any you or any other open group moderators would see as a priority to be accessible on mobile?

It's an absolute priority, yes, and I can illustrate this with a real-world example occurring as I actually write this.

The Privacy group is undergoing a spam attack right at this moment. The only moderator of that group who is on-line right now is an Android-only user. His Android client is barely responding due to the flood of messages, and he has no way to add me as a moderator, because the Android client doesn't offer that as an option.

So, all we can both do right now is watch the messages pile in, unable to delete any of them.

It's a pretty serious shortcoming when a mod cannot add another user as a mod.
